packetWin7/npf/npf/Read.c

```diff
@@ -1026,13 +1026,13 @@ NPF_TapExForEachOpen(
 						{
 							//the Radiotap header will be fragmented in the buffer (aka, it will skip the buffer boundary)
 							ToCopy = Open->Size - LocalData->P;
-							NdisMoveMappedMemory(LocalData->Buffer + LocalData->P, Dot11RadiotapHeader, ToCopy);
-							NdisMoveMappedMemory(LocalData->Buffer + 0, Dot11RadiotapHeader + ToCopy, Dot11RadiotapHeaderSize - ToCopy);
+							RtlCopyMemory(LocalData->Buffer + LocalData->P, Dot11RadiotapHeader, ToCopy);

```

Comment:
This appears already to have been done in the tip of the master branch, so it should no longer be necessary (`NdisMoveMappedMemory()` is no longer being called).